addons/goggles/XEH_postInit.sqf

```diff
@@ -87,13 +88,13 @@ player addEventHandler ["Killed",{
         [GVAR(EyesDamageScript)] call CALLSTACK(cba_fnc_removePreFrameHandler);
     };
     if (GVAR(DustHandler) != -1) then {
-        [GVAR(DustHandler)] call CALLSTACK(cba_fnc_removePerFrameHandler);
+        [GVAR(DustHandler)] call CALLSTACK(CBA_fnc_removePerFrameHandler);
         GVAR(DustHandler) = -1;
     };
 }];
-player addEventHandler ["Fired",{[_this select 0, _this select 1] call FUNC(dustHandler);}];
-player AddEventHandler ["Take",{call FUNC(checkGlasses);}];
-player AddEventHandler ["Put", {call FUNC(checkGlasses);}];
+player addEventHandler ["Fired", DFUNC(dustHandler)];
+player addEventHandler ["Take", DFUNC(checkGlasses)];
+player addEventHandler ["Put", DFUNC(checkGlasses)];
```

Comment:
This stores a whole copy of `DFUNC(blah)` instead of `{_this call Pointer_to_blah}` only.